mimic/model/nova_objects.py

```diff
@@ -527,6 +527,35 @@ def fail_later(server):
     return fail_later
 
 
+@server_creation.declare_behavior_creator("sequence")
+def sequence(parameters):
+    """
+    Sometimes a sequence of behaviors occur when you try to create a server in
+    a predictable pattern.
+
+    Takes one parameter, ``behaviors``, which is a list of specifications of
```

Comment:
Would it make sense to document here that like the list is a dictionary containing the name under the key 'name' and the parameters under the name 'parameters'?  This specification is different than the other behavior specifications (since they just take a dictionary as parameters?)